src/main/resources/assets/paths/collectionsByQuery.yaml

```diff
@@ -0,0 +1,45 @@
+get:
+  summary: Get DAR Collection By Query
+  description: Returns DAR collections that meet query criteria
+  tags:
+    - DAR Collection
+  parameters:
+    - name: filterTerm
+      in: query
+      description: String value that represents term(s) to search upon
+      schema:
+        type: string
+    - name: sortField
+      in: query
+      description: String value that represents field to order by
+      required: true
+      schema:
+        type: string
+    - name: sortOrder
+      in: query
+      description: String value that represents direction of ordering
+      required: true
+      schema:
+        type: string
+    - name: pageSize
+      in: query
+      description: Integer value that represents number of records returned on each page
+      schema:
+        type: integer
+      default: 10
+  responses:
+    200:
+      description: Returns a single DAR Collection
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/DarCollection.yaml'
+    400:
+      description: Bad Request
+    401:
+      description: Unauthorized
```

Comment:
In general, we can ignore these for individual paths. This is true of any `/api/*` call